mod-source-record-manager-server/src/main/resources/templates/db_scripts/update_set_file_name_from_null_to_no_file_name.sql

```diff
@@ -0,0 +1,6 @@
+update diku_mod_source_record_manager.job_execution SET file_name = 'No file name' WHERE file_name IS NULL;
```

Comment:
```suggestion
update ${myuniversity}_${mymodule}.job_execution SET file_name = 'No file name' WHERE file_name IS NULL;
```
diku is the name of the tenant, script should be applicable for all tenants